README.md

```diff
@@ -41,12 +41,13 @@ SuperCollider has guaranteed support for:
 - Windows 10
 - MSVC 2017, 2019
 - macOS 10.14-10.15
-- Xcode 10, 11
+- Xcode 11, 12
```

Comment:
Do we want to change this now?
Currently:
- we build legacy build with XCode 10
- XCode 12 works fine, but only on Intel; I've encountered signing issues on Arm (I haven't submitted an issue though, needs more testing)

Reply:
I can add back 10. I would like to keep 12 because that is what we guarantee according to our own policies, and it's going to be the case for 3.12.0 anyway.